README.md

```diff
@@ -72,10 +73,29 @@ Responses will be mocked using realistic data that conforms to the type in the d
 This will run a proxy server with validation enabled according to given spec.
 
 ```bash
-$ prism server examples/petstore.json
+$ prism server examples/petstore.oas2.json
 > http://127.0.0.1:4010
 ```
 
+Now let's call our server with a non-existing url to see validation in action:
```

Comment:
Is this validation? If so, what type of validation, path validation? 

I think we might be overusing the term validation, maybe we could say "to see what happens when you specify an invalid route"